godot-core/src/builtin/vector2.rs

```diff
@@ -85,6 +81,206 @@ impl Vector2 {
+    }
+
+    pub fn is_finite(self) -> bool {
+        self.to_glam().is_finite()
+    }
+
+    pub fn is_normalized(self) -> bool {
+        self.to_glam().is_normalized()
+    }
+
+    pub fn is_zero_approx(self) -> bool {
+        is_zero_approx(self.x) && is_zero_approx(self.y)
+    }
+
+    pub fn length_squared(self) -> f32 {
+        self.to_glam().length_squared()
+    }
+
+    pub fn lerp(self, to: Self, weight: f32) -> Self {
+        Self::from_glam(self.to_glam().lerp(to.to_glam(), weight))
+    }
+
+    pub fn limit_length(self, length: Option<f32>) -> Self {
+        Self::from_glam(self.to_glam().clamp_length_max(length.unwrap_or(1.0)))
+    }
+
+    pub fn max_axis_index(self) -> i32 {
+        let me = self.to_glam().max_element();
+        (me == self.y) as i32
+    }
+
+    pub fn min_axis_index(self) -> i32 {
+        let me = self.to_glam().min_element();
+        (me == self.y) as i32
+    }
+
+    pub fn move_toward(self, to: Self, delta: f32) -> Self {
+        let vd = to - self;
+        let len = vd.length();
+        if len <= delta || len < CMP_EPSILON {
+            to
+        } else {
+            self + vd / len * delta
+        }
+    }
+
+    pub fn orthogonal(self) -> Self {
+        Self::new(self.y, -self.x)
+    }
+
+    pub fn posmod(self, pmod: f32) -> Self {
+        Self::new(fposmod(self.x, pmod), fposmod(self.y, pmod))
+    }
+
+    pub fn posmodv(self, modv: Self) -> Self {
+        Self::new(fposmod(self.x, modv.x), fposmod(self.y, modv.y))
+    }
+
+    pub fn project(self, b: Self) -> Self {
+        Self::from_glam(self.to_glam().project_onto(b.to_glam()))
+    }
+
+    pub fn reflect(self, normal: Self) -> Self {
+        Self::from_glam(self.to_glam().reject_from(normal.to_glam()))
+    }
+
+    pub fn round(self) -> Self {
+        Self::from_glam(self.to_glam().round())
+    }
+
+    pub fn sign(self) -> Self {
+        -self
+    }
+
+    pub fn slerp(self, to: Self, weight: f32) -> Self {
+        let start_length_sq = self.length_squared();
+        let end_length_sq = to.length_squared();
+        if start_length_sq == 0.0 || end_length_sq == 0.0 {
+            return self.lerp(to, weight);
+        }
+        let start_length = start_length_sq.sqrt();
+        let result_length = lerp(start_length, end_length_sq.sqrt(), weight);
+        let angle = self.angle_to(to);
+        self.rotated(angle * weight) * (result_length / start_length)
+    }
+
+    pub fn slide(self, normal: Self) -> Self {
+        self - normal * self.dot(normal)
+    }
+
+    pub fn snapped(self, step: Self) -> Self {
+        Self::new(snapped(self.x, step.x), snapped(self.y, step.y))
+    }
+
+    /// Returns the result of rotating this vector by `angle` (in radians).
+    pub fn rotated(self, angle: f32) -> Self {
+        Self::from_glam(glam::Affine2::from_angle(angle).transform_vector2(self.to_glam()))
+    }
+
+    #[cfg(not(any(gdext_test, doctest)))]
```

Comment:
The `#[cfg(not(any(gdext_test, doctest)))]` is no longer necessary (as of today, if you rebase on `master`) 🙂